website/docs/workloads/specview/specview-profiles.md

```diff
@@ -0,0 +1,42 @@
+# SPECviewperf Workload Profiles
+The following profile runs the SPECviewperf Workloads.
+
+* [Workload Details](./specview.md)  
+
+## PERF-GPU-SPECVIEW-AMD.json
+Runs the stock SPECviewperf Workloads.
+
+* [Workload Profile](https://github.com/microsoft/VirtualClient/blob/main/src/VirtualClient/VirtualClient.Main/profiles/PERF-GPU-SPECVIEW-AMD.json) 
+
+* **Supported Platform/Architectures**
+  * win-x64
+  * AMD v620 GPU
```

Comment:
Have you tried MI25 and other AMD GPU? ALso would it work on Nvidia GPU?

Reply:
Those are good points. We will add support for Mi25 as well as Nvidia GPU in future PRs.